account_journal_lock_date/views/account_journal.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2017 ACSONE SA/NV
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <data>
+        <record model="ir.ui.view" id="account_journal_form_view">
```

Comment:
The ID does not conform to the proper naming. (nor does the Odoo core XML ID for that matter)
According to the guidelines the original XML ID should be used: https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#253inherited-xml